test/unit/controllers/search.js

```diff
@@ -17,14 +18,89 @@ describe('controllers/search', function () {
   var api = {
     read: sinon.stub()
   };
+  var hof = require('../../../lib/hof-standalone');
   var searchController = proxyquire('../../../controllers/search', {
-    '../api': api
+    '../api': api,
+    '../lib/hof-standalone': hof
   });
 
   it('is a function', function () {
     searchController.should.be.a('function');
   });
 
+  describe('validator', () => {
+    describe('british-date', () => {
+      it('should exist as a function', () => {
+        hof.validators.should.have.property('british-date').that.is.a('function');
+      });
+
+      it('should accept properly formatted dates', () => {
+        hof.validators['british-date']('31/12/1999').should.be.ok;
+        hof.validators['british-date']('3/1/1999').should.be.ok;
+        hof.validators['british-date']('3/1/99').should.be.ok; // really!?
+        hof.validators['british-date']('1/1/1').should.be.ok; // really!? really!?
```

Comment:
I vote leave it as is for now, because it at least documents the unwanted behavior... then when it's fixed properly they can be moved to the **invalid** test case instead.